Implement breadcrumb menu in all admin pages

andreslucena opened the pull request:
#### :tophat: What? Why?

There were some pages without breadcrumb in the admin. This PR fixes it.

Apart from that, I found a couple mini bugs:

- We were using the `components-counter--off` in the HTML but it didn't have any CSS 
- The components counter in the Global Moderations is broken, as it didn't correspond to the ones that we have in components. (Just thinking out loud that we should rename it to `resources-counter` or something more generic?)

#### :pushpin: Related Issues

- Fixes #11387

#### Testing

1. Visit different pages in the admin panel
2. See that the breadcrumb corresponds to the page that you're at 

:hearts: Thank you!

alecslupu reviewed (changes requested):
@andreslucena  the PR works as expected, however, the following does not have the correct breadcrumbs: 

- [x] Votings -> Monitoring committee
- [x] Participatory Process -> Categories 
- [x] Participatory Process -> Categories 
- [ ] Pages -> Topics ( is displaying the breadcrumb as "Pages") 
- [ ] Participants -> Identity documents
- [ ] Participants -> Code by postal letter
- [ ] Participants -> Organization's census

Also, i am wondering if the Attachment section should display "Files" / "Folders" areas in breadcrumbs ( see any participatory space for reference).

alecslupu commented:
Apart of the things i have found, I am also for renaming `components-counter` to whatever more generic.

andreslucena commented:
> @andreslucena the PR works as expected, however, the following does not have the correct breadcrumbs:
> 
>     * Votings -> Monitoring committee

Done on 88664edb17a90d989cb3ae5ea72d384463d58ed2 
> 
>     * Participatory Process -> Categories
> 
>     * Participatory Process -> Categories

Fixed on 913f8e079756510422bcc63e833a951865c39ecc

> 
>     * Pages -> Topics ( is displaying the breadcrumb as "Pages")

Done on 06c1c02d47a0f4dde38c4f3dd927011ed956c0c2
> 
>     * Participants -> Identity documents
> 
>     * Participants -> Code by postal letter
> 
>     * Participants -> Organization's census

Done on 9747b5ad33a4729afd642f9fc82260cd41008360
> 
> 
> Also, i am wondering if the Attachment section should display "Files" / "Folders" areas in breadcrumbs ( see any participatory space for reference).

I don't see it clear neither. This is how it'd look like:

![Screenshot of the attaments breadcrumb with the Files as the last element](https://github.com/decidim/decidim/assets/717367/3088c74b-5513-4fb2-bff3-9a4a670a4a9f)

I say let's merge this as is for now. We'll talk about this in the next @decidim/product meeting and if we agree that this should be here, then I'll open the PR. (cc/ @carolromero)

alecslupu reviewed (dismissed):
While reviewing i have noticed that : 

- In votings you have added "Monitoring Committee" as breadcrumb part

While in the Authorization menu, you have not added that to the breadcrumb menu fragment.

- Participants -> Identity documents
- Participants -> Code by postal letter
- Participants -> Organization's census

Also, the Organization census, is displaying the name with html entities  (ex: `Organization&#39;s census`)

![image](https://github.com/decidim/decidim/assets/105683/fcff1e71-5e29-4417-9e7c-e492474b84df)

Can you have a look ?

carolromero commented:
> I don't see it clear neither. This is how it'd look like:
> 
> ![Screenshot of the attaments breadcrumb with the Files as the last element](https://github.com/decidim/decidim/assets/717367/3088c74b-5513-4fb2-bff3-9a4a670a4a9f)
> 
> I say let's merge this as is for now. We'll talk about this in the next @decidim/product meeting and if we agree that this should be here, then I'll open the PR. (cc/ @carolromero) 

Yep, let's merge like this. The breadcrumb is already consistent with the side menu. Depending on how we end up settling this menu we will see if we need to add it in the future.

alecslupu commented:
> Yep, let's merge like this. The breadcrumb is already consistent with the side menu. Depending on how we end up settling this menu we will see if we need to add it in the future.

At the moment, what @andreslucena presented in the image is not yet implemented. Also, before merging i would address the comments frommy previous [comment](https://github.com/decidim/decidim/pull/12197#pullrequestreview-1803785469 )

carolromero commented:
@alecslupu yes Alex, I meant not implement it for now (sorry that I copy/pasted the screenshot and it looked confusing)

andreslucena commented:
> While in the Authorization menu, you have not added that to the breadcrumb menu fragment. 

Done at 532c7339fdc007971a47571c91e0f1d4055dd8c2
I've also fixed the same problem with the Impersonations menu (see 2910b5adf71771109a78dbc1fb613d5997323b07)  

> Also, the Organization census, is displaying the name with html entities (ex: `Organization&#39;s census`)

Fixed in 8d8acb54e7cddd1b395bd4a24b72ad79cd724416 

---

On another hand, @carolromero has asked me about the "Scope types" and "Area types" pages, that weren't with breadcrumb (nor sidebar active link). These were solved with d901ac84e1912180476fa4a389a2b1482e16ec17 and afe475edf67344d273ef8d87ba2b5d013901601b 

---

This is ready for another round @alecslupu

andreslucena commented:
> This is ready for another round @alecslupu

![Morpheus in Matrix ready for another round](https://github.com/decidim/decidim/assets/717367/1d48de42-31d0-454d-aa6f-627c18d6cf5a)

alecslupu reviewed (approved):
:+1:

alecslupu commented:
adding no-backport, as this is a redesign ticket, that should not reach into 0.27.
